spec/support/request_helpers.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
+
 include Warden::Test::Helpers
 
 module RequestHelpers
   def login(user = nil, admin: false, operator: false)
     user ||= FactoryGirl.create(:user)
 
-    case
-    when admin
+    if admin
       FactoryGirl.create(:user_role, :admin, user: user)
-    when operator
+    elsif operator
```

Comment:
__[Reek]__

[ControlParameter] RequestHelpers#login is controlled by argument 'operator'


<!-- Comment by [SideCI](https://sideci.com) -->